css/login.css

```diff
@@ -0,0 +1,120 @@
+body{
+	margin: 0;
+
+	background-color: #555555;
+	font-family: 'Roboto', sans-serif;
+}
+
+.section{
+	height: 640px;
+}
+
+.section1{
```

Comment:
Both sections have something in common. That's .section. 
.Section1 and .Section2 different styles.